### PR TITLE
new houserush sample framework

### PR DESCRIPTION
**Pull request by alec-z**
Add the new e-commerce sample project framework as my proposal says. It's just the pom files and the microservices split design. @liubao68

### CODE REVIEW

**Comment by yhs0092** [comment]
Hello, I find that the indent of some files is different from the others. It's recommended to use the same code format template, and you can find it in [ServiceComb-Java-Chassis](https://github.com/apache/servicecomb-java-chassis/tree/master/etc).
And could you add some introduction into the README file, such as the backgroud introduction and how to run it?

**Comment by alec-z** [comment]
@yhs0092 I have 
reformat code with google java code style and add some Readme content

File: houserush/doc/pom.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (WillemJiang) [inline houserush/doc/pom.xml]: We need to add Apache License.

File: houserush/house-order/pom.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>mysql</groupId>
```
Comment (WillemJiang) [inline houserush/house-order/pom.xml]: We cannot use the mysql connector due to the license issue, please switch it to postgresql.

File: houserush/pom.xml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    </dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-maven-plugin</artifactId>
+          <version>2.0.1.RELEASE</version>
```
Comment (WillemJiang) [inline houserush/pom.xml]: You may need to consider to upgrade the spring boot version, as this one is pretty old.

File: houserush/house-order/pom.xml (hunk 2 above)
Comment (alec-z) [reply houserush/house-order/pom.xml]: I will change it, thx.

File: houserush/pom.xml (hunk 3 above)
Comment (alec-z) [reply houserush/pom.xml]: OK

**Pull request merged**